examples/README.md

```diff
@@ -318,13 +318,12 @@ FABRIC_SCRIPTDIR="${FPC_TOP_DIR}/fabric/bin/"
 . ${FABRIC_SCRIPTDIR}/lib/common_utils.sh
 . ${FABRIC_SCRIPTDIR}/lib/common_ledger.sh
 
-CC_ID=helloworld_test
-CC_EP="OR('SampleOrg.member')"
-
 #this is the path that will be used for the docker build of the chaincode enclave
-ENCLAVE_SO_PATH=examples/helloworld/_build/lib/
```

Comment:
This ENCLAVE_SO_PATH is still used in several places below. Also, as the same is used in `peer.sh`, I would leave it.

Reply:
good catch! fixed